mapeditor/mapcontroller.h

```diff
@@ -64,6 +67,8 @@ class MapController
 	MainWindow * main;
 	mutable std::array<std::unique_ptr<MapScene>, 2> _scenes;
 	mutable std::array<std::unique_ptr<MinimapScene>, 2> _miniscenes;
+	std::vector<std::unique_ptr<CGObjectInstance>> _clipboard;
```

Comment:
Huh, why is editor uses different naming style? I see that some of these were before, but we generally don't use underscore for class members.

Reply:
It's more pythonic way to distinguish private members

Reply:
...but this is c++, vcmi. We use different style.

Reply:
Qt, fl-static for example use same approach, so it's quite common naming in c++. If you think it's something really important, I can change, but better in another PR to avoid quite large irrelevant change

Reply:
I'm not saying that this is bad style, but this is not style we use in all other projects, including Qt-based Launcher and FL-based AI's

I'll agree with "outside of PR scope" argument, but in long-term I think that it should be switched to one we use everywhere else.